### PR TITLE
code coverage

### LINKED ISSUE #42: code coverage for outrigger
add code coverage checking to outrigger so we can figure out where the testing gaps currently are.

### PR DESCRIPTION
**Pull request by rlgomes**
adding code coverage checking for outrigger

fixes #42

### CODE REVIEW

**Comment by rlgomes** [comment]
@mstemm @go-oleg @mnibecker @demmer code coverage for outrigger. The numbers are not fantastic at this point but follow up PRs will come to fill in those gaps.

File: gulpfile.js [hunk 1]
```diff
@@ -54,6 +51,20 @@ gulp.task('jshint-lib', function() {
 
 gulp.task('lint', ['jscs-lib', 'jscs-test', 'jshint-lib', 'jshint-test']);
 
+gulp.task('instrument', function () {
+    return gulp.src([
+        'lib/**/*.js',
+        'src/**/*.js'
+        // XXX: bin/demo bin/client ?
```
Comment (mstemm) [inline gulpfile.js]: I don't think these exist any longer, do they?

Comment (mstemm) [inline gulpfile.js]: Oh but you do want to add bin/outriggerd and bin/outrigger-client.

**Comment by mstemm** [comment]
What's isparta used for? I don't see it used in the gulpfile.

**Pull request merged**